Sources/StreamChat/Config/ChatClientConfig.swift

```diff
@@ -112,18 +112,35 @@ public struct ChatClientConfig {
     /// - Parameter apiKey: The API key of the chat app the `ChatClient` connects to.
     ///
     
-    /// Allows to inject a custom API client for uploading attachments, if not specified `StreamCDNClient` is used
+    /// Allows to inject a custom API client for uploading attachments, if not specified, `StreamCDNClient` is used.
     public var customCDNClient: CDNClient?
+
+    /// Allows to inject a custom attachment uploader. It can be used to have more
+    /// control than `CDNClient` to allow changing the attachment payload.
+    public var customAttachmentUploader: AttachmentUploader?
```

Comment:
Maybe we should specify that if this one is used, the `customCDNClient` above doesn't have any effect?

Reply:
Yes make sense. Also good to say this in docusaurud